src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColorSupplier randomColor = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        Name[] names = Name.values();
+        int index = random.nextInt(names.length);

```

Comment:
```suggestion
        int index = random.nextInt(names.length);
        String randomColor = randomColor.getRandomColor();
```

provide to all figures